icu4c/source/common/uresdata.cpp

```diff
@@ -33,6 +33,7 @@
 #include "uinvchar.h"
 #include "uresdata.h"
 #include "uresimp.h"
+#include "utracimp.h"
```

Comment:
We no longer limit ourselves to 8.3 source code filenames. Please use utraceimp.h or utrace_imp.h or similar.